contrib/helm/kind-cluster.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    kubeadmConfigPatches:
+      - |
+        kind: InitConfiguration
+        nodeRegistration:
+          kubeletExtraArgs:
+            node-labels: "ingress-ready=true"
+    extraPortMappings:
+      - containerPort: 80
```

Comment:
Should we open up 443 to the kind cluster as well in case users want to test the ingress with TLS in the cluster?

Reply:
I wanted to as least invasive as possible on the host machine of someone trying this out. I can add 443 as well if you think it makes sense. @JoelSpeed thoughts?